app.py

```diff
@@ -3,33 +3,29 @@
 import os
 import json
 from pathlib import Path
-from flask import Flask, render_template
+from flask import Flask
+
+# Importing Modules
+import helper_functions as f
 
 app = Flask(__name__) #creating Flask class object
 title='Quotation'
 
 @app.route('/') #decorator defines the
 @app.route('/home')
-@app.route('/base')
 @app.route('/index')
 @app.route('/homepage')
```

Comment:
Too many URLs are pointing to the single functions. Earlier this was a part of HTML but not now. We can just keep a single URL for the purpose and let handle it from the Web UI.